src/features/groups/components/GroupCardList/GroupCard/GroupCardContent/GroupCardDropdownMenuTriggerButton/index.tsx

```diff
@@ -2,25 +2,29 @@
 import { MenuKebabIcon } from '@/assets/images/icons';
 import { Button, DropdownMenu, DropdownMenuTrigger, Icon } from '@/components/ui';
 
-import { FC } from 'react';
+import { useState } from 'react';
 
 import { GroupCardDropdownMenuContent } from './GroupCardDropdownMenuContent';
 
 interface IGroupCardMenuButtonProps {
   handleRenameClick: () => void;
 }
 
-export const GroupCardDropdownMenuTriggerButton: FC<IGroupCardMenuButtonProps> = ({
+export const GroupCardDropdownMenuTriggerButton = ({
   handleRenameClick,
-}) => {
+}: IGroupCardMenuButtonProps) => {
+  const [isDropdownMenuOpen, setIsDropdownMenuOpen] = useState(false);
   return (
-    <DropdownMenu>
+    <DropdownMenu open={isDropdownMenuOpen} onOpenChange={setIsDropdownMenuOpen}>
       <DropdownMenuTrigger asChild>
         <Button variant="ghost" className="w-12">
           <Icon icon={MenuKebabIcon} size={4.5} />
         </Button>
       </DropdownMenuTrigger>
-      <GroupCardDropdownMenuContent handleRenameClick={handleRenameClick} />
```

Comment:
Because I separated `GroupCardDropdownMenuContent` from trigger as a different component, 
I ended up having to pass `onDropdownMenuClose` prop from this file, and it's props drilling among 4 layers.

(`GroupCardDropdownMenuTriggerButton` => `GroupCardDropdownMenuContent` => `DeleteGroupDialogTriggerButton` => `DeleteGroupDialogContent`)

The directory structure itself is reasonable one tho, let me know if you feel we have to do something for this props drilling. I personally chose to keep this directory structure for now.

Reply:
> I personally chose to keep this directory structure for now.

I think it's acceptable for now, too.